functions/paymentCollector/src/index.ts

```diff
@@ -0,0 +1,88 @@
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { marshall } from "@aws-sdk/util-dynamodb";
+import { DynamoDBDocumentClient, QueryCommand, UpdateCommand } from '@aws-sdk/lib-dynamodb';
+import type { Event } from "./event";
+
+const client = new DynamoDBClient({ region: "us-west-2" });
+const ddb = DynamoDBDocumentClient.from(client);
+
+export const handler = async (event: Event) => {
+  const body = event.body
+  const tableName = event.environment === "prod" ? "paymentJobs-prod" : "paymentJobs"
+  let message = "Unknow status"
+  let statusCode: number | undefined = 200
+
+  if (!await paymentExist(body.pk, body.sk, tableName)) {
+    throw new Error("Payment does not exisist!")
```

Comment:
What about if we have error codes? 

I'm thinking that would be very nice to have a centralized error code library or something like that consumed by the UI and the API so we ensure the same error is shared across the stack, any thoughts?

Reply:
I agree